lua/plenary/busted.lua

```diff
@@ -238,17 +219,50 @@ clear = mod.clear
 assert = require("luassert")
 
 mod.run = function(file)
+  print("\n" .. HEADER)
+  print("Testing: ", file)
```

Comment:
Just thinking out load right now. We could do `Path:normalize` so we are going from `/home/conni/repos/neovim/plenary.nvim/tests/plenary/simple_busted_spec.lua` to `tests/plenary/simple_busted_spec.lua`. :)

Reply:
Yeah, feel free to make another PR to tidy up any display things, since that would be nice I think. You can merge in whenever